x-pack/plugins/uptime/public/state/api/alerts.ts

```diff
@@ -131,11 +131,20 @@ export const fetchAlertRecords = async ({
     sort_field: 'name.keyword',
     sort_order: 'asc',
   };
-  const alerts = await apiService.get<{ data: Array<Alert<NewAlertParams>> }>(
+  const rawRules = await apiService.get<{ data: Array<Alert<NewAlertParams>> }>(
     API_URLS.RULES_FIND,
     data
   );
-  return alerts.data.find((alert) => alert.params.monitorId === monitorId) as Alert<NewAlertParams>;
+  const monitorRule = rawRules.data.find(
+    (rule) => rule.params.monitorId === monitorId
+  ) as Alert<NewAlertParams>;
```

Comment:
Is this type not changing?

Reply:
You are correct, but I do think we have an issue here and we should expose a RuleClientUI so we all talk the same type. I am going to create an issue to create a RuleAPI on the UI side.